util/db/cluster.go

```diff
@@ -233,17 +233,17 @@ func (db *db) GetCluster(_ context.Context, server string) (*appv1.Cluster, erro
 func (db *db) GetProjectClusters(ctx context.Context, project string) ([]*appv1.Cluster, error) {
 	informer, err := db.settingsMgr.GetSecretsInformer()
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed to get secrets informer: %w", err)
 	}
 	secrets, err := informer.GetIndexer().ByIndex(settings.ByProjectClusterIndexer, project)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed to get index by project cluster indexer: %w", err)
```

Comment:
nit: Maybe add the error message to specify which project?

Reply:
done!